### PR TITLE
Update OCP-14799 to inactive

### PR DESCRIPTION
**Pull request by weinliu**
Update OCP-14799 to inactive for now, debugging sill need more time, it will be covered in manual run before fixed.

### CODE REVIEW

**Comment by weinliu** [comment]
@liangxia could you help? Thanks

File: features/infrastructure/cluster-capacity.feature [hunk 1]
```diff
@@ -4,7 +4,7 @@ Feature: cluster-capacity related features
   # @author weinliu@redhat.com
   # @case_id OCP-14799
   @admin
-  @flaky
+  @inactive
```
Comment (liangxia) [inline features/infrastructure/cluster-capacity.feature:7]: `flaky` for short term, while `inactive` for long term.
Based on the description, sounds like you still need the test, just wan to disable it for a short time.

Comment (weinliu) [reply features/infrastructure/cluster-capacity.feature:7]: yes, `flaky` will have the case go to waiting state in the test run, and the team got pinged frequently, so prefer to have it being `inactive` for a while

Comment (liangxia) [reply features/infrastructure/cluster-capacity.feature:7]: Change to inactive in code does not resolve the issue, as test case selection is done via interacting with Polarion.
If you would like the test case not picking up in Jenkins CI for some time, it's suggested to move the test case into "not automated" status in Polarion.

**Comment by jhou1** [comment]
I'm receiving a daily notification to approve this, so, @weinliu it looks like is pending your decision.

`@inactive` is when you no longer want this test performed. As Liang's comment, you need to make your test case inactive too, otherwise you are taking a risk your test is automated but you don't have it actually running.
`@flakly` is when you want to work with a PR to fix the automation later and decide to temporarily disable the test.

Let us know your choice, thanks.

**Comment by weinliu** [comment]
Resolved in another PR. Thanks

**Pull request closed without merging**